app/src/Entities/Capture.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Audubon\Entities;
+
+class Capture
+{
+    private $id;
+    private $date_time;
+    private $location;
+    private $member_id;
```

Comment:
In most cases, do not use _.  So it should be $memberID.  $date_time could be $date or something more descriptive if you want.  Generally $date is pretty obvious what it is for.  Like in this case it's safe to assume that date is the date of the capture, so I don't think it's necessary to be any more descriptive than $date.